playbilling/src/test/java/com/google/androidbrowserhelper/playbilling/digitalgoods/DigitalGoodsTests.java

```diff
@@ -160,7 +161,8 @@ public void acknowledgeCall_callsConsume() throws InterruptedException {
     private void callAcknowledge(boolean makeAvailableAgain) throws InterruptedException {
         Bundle args = AcknowledgeCall.createBundleForTesting("id1", makeAvailableAgain);
         CountDownLatch callbackTriggered = new CountDownLatch(1);
-        int expectedResponseCode = 23;
+        int responseCode = BillingClient.BillingResponseCode.ITEM_NOT_OWNED;
```

Comment:
Perhaps clearer variable names would be billingResponseCode and chromiumResponseCode?